### PR TITLE
Configure jest and add tests for interpolate.

### PR DESCRIPTION
**Pull request by DylanVann**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
   "name": "react-native-reanimated",
   "version": "1.0.0-alpha.3",
-  "description": "More powerfull alternative to Animated library for React Native",
+  "description": "More powerful alternative to Animated library for React Native.",
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
-    "test": "jest",
-    "format": "prettier --write './src/**'",
-    "lint": "eslint --fix './src/**'",
+    "test": "npm run format && npm run lint && npm run test:unit",
```
Comment (DylanVann) [inline package.json]: `test` just checks everything.

File: src/ReanimatedEventEmitter.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import ReanimatedModule from './ReanimatedModule';
+import { NativeEventEmitter } from 'react-native';
+
+export default new NativeEventEmitter(ReanimatedModule);
```
Comment (DylanVann) [inline src/ReanimatedEventEmitter.js]: Only using one `NativeEventEmitter`, maybe that's a good thing.

**Comment by DylanVann** [comment]
@kmagiera done

**Comment by kmagiera** [comment]
Thanks so much just 🎉🎉

**Pull request merged**